shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -1031,6 +1031,10 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
     result(@{@"value" : @([self clipboardHasStrings])});
   } else if ([call.method isEqualToString:@"System.exitApplication"]) {
     [[self terminationHandler] handleRequestAppExitMethodCall:call.arguments result:result];
+  } else if ([call.method isEqualToString:@"System.initializationComplete"]) {
+    // TODO(chrome-bot): Handle this message to enable exit message listening.
```

Comment:
Is `chrome-bot` something I don't yet know about or should this be gspencergoog?

Reply:
`chrome-bot` was from the patch generated by the lint failure. It looks like no one is assigned to the open linux PR, but he authored it, so probably.